src/plots/plot.ts

```diff
@@ -479,11 +495,14 @@ export class Plot extends Component {
     let dataToDraw = this._getDataToDraw();
     let drawers = this._getDrawersInOrder();
 
-    this.datasets().forEach((ds, i) => drawers[i].draw(dataToDraw.get(ds), drawSteps));
+    // this.datasets().forEach((ds, i) => drawers[i].draw(dataToDraw.get(ds), drawSteps));
+    //
+    // let times = this.datasets().map((ds, i) => drawers[i].totalDrawTime(dataToDraw.get(ds), drawSteps));
+    // let maxTime = Utils.Math.max(times, 0);
+    // this._additionalPaint(maxTime);
```

Comment:
i don't know what "additionalPaint" is or why it has a time argument. Is this difficult to abstract?